--- nifti_clib-3.0.0/CMakeLists.txt.orig	2024-04-27 17:42:15.648355337 +0200
+++ nifti_clib-3.0.0/CMakeLists.txt	2024-04-27 17:46:23.020348541 +0200
@@ -69,7 +69,7 @@ if(BUILD_SHARED_LIBS AND NOT NIFTI_LIBRA
 endif()
 
 set_if_not_defined(NIFTI_INSTALL_RUNTIME_DIR bin)
-set_if_not_defined(NIFTI_INSTALL_LIBRARY_DIR lib)
+set_if_not_defined(NIFTI_INSTALL_LIBRARY_DIR lib${LIB_SUFFIX})
 set_if_not_defined(NIFTI_INSTALL_ARCHIVE_DIR ${NIFTI_INSTALL_LIBRARY_DIR})
 set_if_not_defined(NIFTI_INSTALL_INCLUDE_DIR include/nifti)
 set_if_not_defined(NIFTI_INSTALL_MAN_DIR share/man/man1)
